coins/src/adapters/tokenMapping_added.json

```diff
@@ -8037,6 +8037,18 @@
       "decimals": "18",
       "symbol": "suBTC",
       "to": "coingecko#bitcoin"
+    },
+    "0x5a2aa871954eBdf89b1547e75d032598356caad5": {
+      "name": "pumpBTC",
+      "decimals": "8",
+      "symbol": "pumpBTC",
+      "to": "coingecko#pumpbtc"
+    },
+    "0x8BB97A618211695f5a6a889faC3546D1a573ea77": {
+      "name": "nBTC",
+      "decimals": "8",
+      "symbol": "nBTC",
+      "to": "coingecko#bitcoin"
```

Comment:
this is still the wrong file - this is tokenMapping_added.json. Please use tokenMapping.json